stdlib/public/libexec/swift-backtrace/TargetMacOS.swift

```diff
@@ -260,24 +266,32 @@ class Target {
         exit(1)
       }
 
-      guard let symbolicated = backtrace.symbolicated(with: images,
-                                                      sharedCacheInfo: sharedCacheInfo,
-                                                      useSymbolCache: cache) else {
-        print("unable to symbolicate backtrace from context for thread \(ndx)",
-              to: &standardError)
-        exit(1)
-      }
+      if symbolicate {
+        guard let symbolicated = backtrace.symbolicated(with: images,
+                                                        sharedCacheInfo: sharedCacheInfo,
+                                                        useSymbolCache: cache) else {
+          print("unable to symbolicate backtrace from context for thread \(ndx)",
+                to: &standardError)
+          exit(1)
+        }
 
-      threads.append(TargetThread(id: info.thread_id,
-                                  context: ctx,
-                                  name: threadName,
-                                  backtrace: symbolicated))
+        threads.append(TargetThread(id: info.thread_id,
+                                    context: ctx,
+                                    name: threadName,
+                                    backtrace: .symbolicated(symbolicated)))
+      } else {
+        threads.append(TargetThread(id: info.thread_id,
+                                    context: ctx,
+                                    name: threadName,
+                                    backtrace: .raw(backtrace)))
+      }
 
       mach_port_deallocate(mach_task_self_, ports[Int(ndx)])
     }
   }
 
-  public func redoBacktraces(limit: Int?, top: Int, cache: Bool) {
+  public func redoBacktraces(limit: Int?, top: Int,
```

Comment:
No need to do it now, but it might be nice to pull out these options into some sort of options struct so future changes don't have to be repeated all over the place.

Reply:
Yeah, I think the entire thing could do with a tidy up, honestly.

Reply:
I think this is probably a job for a separate PR though.